method_comparison/causal_lm/temporary_results/lora--rank32--2025-02-25T16-39-53+00-00.json

```diff
@@ -0,0 +1,47 @@
+{
+  "run_info": {
+    "created_at": "2025-02-25T16:39:53+00:00",
+    "total_time": 7.109975468978519,
+    "experiment_name": "lora/rank32",
+    "peft_branch": "ben-method-comparison",
+    "train_params_sha": "c9983ccde8233a74c1d67126e45cf43440cde83a224cfe9f9aac89136715489c",
+    "peft_config_sha": "3b1a995103b79579ce90b7d46b2138bfb070d60281cdfbdc91d6cbddaaf7e939"
+  },
+  "train_info": {
+    "cuda_memory_avg": 2650153676,
+    "cuda_memory_max": 9621824000,
+    "cuda_memory_90th": 2738772070,
+    "train_time": 2.14472757501062,
+    "file_size": 13640008,
+    "status": "success",
+    "metrics": [],
+    "losses": [
+      5.732011318206787,
+      3.821676254272461,
+      4.096625328063965,
+      4.529172420501709,
+      3.2956929206848145,
+      3.893981456756592,
+      4.66800594329834,
+      5.82646369934082,
+      3.189246892929077,
+      3.522594690322876
+    ]
```

Comment:
parameters are not tracked here - is this expected?

Reply:
You mean the `training_params.json` and `adapter_config.json`? I didn't add them, I only added their hashes.

Reply:
For rendering it might be beneficial to include them. WDYT?

Reply:
Done. I removed the hashes, as I don't think there is a point in having both.